QStreak/View Controllers/Record List/RecordListViewController.swift

```diff
@@ -41,8 +41,8 @@ class RecordListViewController: UIViewController {
     @IBAction private func addRecordBarButtonItemTapped(_ sender: Any) {
         let addRecordStoryboard = UIStoryboard(name: String(describing: AddRecordViewController.self), bundle: nil)
         let addRecordViewController = addRecordStoryboard.instantiateViewController(withIdentifier: String(describing: AddRecordViewController.self))
-        addRecordViewController.presentationController?.delegate = self
-        present(addRecordViewController, animated: true, completion: nil)
+
+        self.navigationController?.pushViewController(addRecordViewController, animated: true)
```

Comment:
[Based on the designs in figma](https://www.figma.com/file/lTpczu51zx7uTIP0aBSSUC/User-Flow?node-id=14%3A372) it seems like the onboarding flow should be a push and that we'll ultimately move from the record creation view to a detail view after the record is created instead of to the list view. That being said, I'm not sure this'll stay like this and am open to changing it back if you think its better that way.

Reply:
Just wondering. You can leave it as is. Be sure to remove `UIAdaptivePresentationControllerDelegate` if you're not going to use that anymore.